src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String decription;
```

Comment:
The variable 'decription' seems to be misspelled. It should probably be 'description'.

Reply:
Done